wallets/rn_cli_wallet/android/app/build.gradle

```
@@ -78,27 +79,45 @@ android {
         applicationId "com.walletconnect.web3wallet.rnsample"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 9
+        versionCode 15
         versionName "1.0"
     }
```

You need to load secrets file to use it for the signingConfigs
```suggestion
    }
    def secretsProperties = new Properties()
    secretsProperties.load(new FileInputStream(rootProject.file("secrets.properties")))
```